mobify/lockup
# Add detect

Replacing custom browser detection code with `detect` library.

Status: Ready to merge

Reviewers: @tedtate @kpeatt 
JIRA: https://mobify.atlassian.net/browse/CSOPS-717
## Changes
- removed custom browser detection code
- added `detect` library
## How to test-drive this PR
- checkout this branch
- `grunt test`
- add this branch to Pinny and test to make sure it works
